### DISCUSSION TITLE
VMWare Operating System?

### DISCUSSION THREAD

**Discussion by Sazerak** [Security-Onion-Solutions/securityonion#11941] [2.4]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

24

### RAM

256

### Storage for /

1TB

### Storage for /nsm

2TB

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I'm about to upgrade from 2.3 to 2.4 and noticed that the docs still say to chose CentOS 64-Bit as the operating system https://docs.securityonion.net/en/2.4/vmware.html (Step 4 of Workstation Pro and Step 4 of Fusion). Is this a mistake? My understanding is that 2.4 is built using Oracle Linux 9 which under the hood is RHEL 9. VMWare products have the option to chose Red Hat Enterprise Linux 9 (64-bit) as an option, so wouldn't that be more appropriate?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
It shouldn't really make any difference whether you choose RHEL 9 or Oracle Linux 9 as long as CPU, RAM, etc. settings are correct. However, I went ahead and updated the documentation to avoid the question in the future:
https://docs.securityonion.net/en/2.4/vmware.html
